README.md

```diff
@@ -1,46 +1,56 @@
+
+Features:
+<ul>
+  <li>
+    <strong>Colour highlighting</strong> - Highlights IES classes according to their defined colours<br>
+    <img src="./images/Overview.png" alt="Highlights IES classes according to their defined colours" style="height:300px">
+  </li>
+  <li>
+    <strong>Diagnostics</strong> - Syntax highlights missing commas, dots, semicolons<br>
+    <img src="./images/Diagnostics.png" alt="Syntax highlights missing commas, dots, semicolons">
+  </li>
+  <li>
+    <strong>IES class suggestions</strong> - Suggests IES classes as you type<br>
+    <img src="./images/Suggestions.jpeg" alt="Suggests IES classes as you type" style="height:300px">
+  </li>
+  <li>
+    <strong>On hover matches</strong> - Hover over a URI to reveal IES type<br>
+    <img src="./images/OnHover.png" alt="Hover over a URI to reveal IES type">
+  </li>
+</ul>
 
-More information about IES can be found here: [IES4 GitHub repository](https://github.com/dstl/IES4)
 
-### Features
 
-- **Colour highlighting:** Highlights IES classes according to their defined colours
-- **Diagnostics:** Syntax highlights missing commas, dots, semicolons
-- **Autocomplete Classes:** Suggests IES classes as you type
-- **On-hover reveal URI type** Hover-over a URI to reveal its type
 
-#### Color highlighting
-![colour highlighting screenshot](./images/Overview.png)
-#### Diagnostics
-![syntax diagnostics screenshot](./images/Diagnostics.png)
-#### IES class suggestions
-![IES class suggestion screenshot](./images/Suggestions.jpeg)
-#### On hover matches
-![On-hover IES type tracking](./images/OnHover.png)
 
 ## Install
 
-1. Download the latest release from: [IES Syntax Highlighter releases](https://github.com/Telicent-io/ies-ext/releases)
+1. Download latest [releases](https://github.com/telicent-oss/ies-syntax-highlighter-vscode-extension/releases)
+
+In VSCode:
 
-2. Bring up the Extensions view by clicking on the Extensions icon in the **Activity Bar** on the side of VS Code or the **View: Extensions** command `Cmd/Ctrl + Shift + x`.
+2. Click "Extensions" icon in the **Activity Bar** on the side of VS Code or the **View: Extensions** command `Cmd/Ctrl + Shift + x`.
 
 3. Click on the Extensions view command dropdown, the three dots (`...`), located at the top of the Extensions view and select **Install from VSIX...**
 
-4. Navigate to the downloaded file and select it.
+4. Navigate to the downloaded release file (step 1) and select it.
 
-5. Select theme IES Highlighter dark+ default `Cmd/Ctrl + k` then `Cmd/Ctrl + t`. Or go to the Extensions view -> ies-extension -> Set Color Theme.
+5. Select theme IES Highlighter <sup>1</sup>dark+ default `Cmd/Ctrl + k` then `Cmd/Ctrl + t`. Or go to the Extensions view -> ies-extension -> Set Colour Theme.
 
+<sup>1</sup> **Note:** Syntax highlighting has not been optimized for "light" themes i.e. highlights `ies:` prefix as _white_
```

Comment:
```suggestion
<sup>1</sup> **Note:** Syntax highlighting has not been optimized for "light" themes i.e. highlights `ies:` prefix as _white_
```
```suggestion
<sup>1</sup> **Note:** Syntax highlighting has not been optimized for "light" themes i.e. highlights `ies:` prefix as _white_
```
```suggestion
<sup>1</sup> **Note:** Syntax highlighting has not been optimized for "light" themes, e.g. highlights `ies:` prefix as _white_
```

Reply:
The back and forth is getting inefficient - lets just merge this, and let you open a PR and drive the changes?

Reply:
That's fine by me.  If @dptelicent is happy I'll make my last tweak in a separate PR once you've closed this one.  

I assume you don't need to review it, @ACoolmanTelicent ?

Reply:
Not me specifically, but I the normal PR process is to get 1 approval

Reply:
No worries, if I do raise one I'll fire it out to the same audience here and see what comes back!